static/styles/message_row.css

```diff
@@ -0,0 +1,239 @@
+            grid-column-start: 2;
+        }
+
+        .alert-msg {
+            grid-row-start: 1;
+            grid-column: 3 / 5;
+            margin-top: 4px;
+        }
+
+        .message_length_controller {
+            grid-row-start: 3;
+            grid-column-start: 2;
+        }
+    }
+
+    &.include-sender {
+        /*
+        1           2                                                                                                    3              4            5
+      1 |‾‾‾‾‾‾‾‾‾‾‾:‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾:‾‾‾‾‾‾‾‾‾‾‾‾‾‾:‾‾‾‾‾‾‾‾‾‾‾‾|
+        |  ((((\\\  : Sender Name  EDITED                                                                                :   +  ⋮  ☆    :  10:00 AM  |
+      2 |   9_9 3)) :_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ : _ _ _ _ _ _ _:_ _ _ _ _ _ |
+        |   \=  ((  :   TEXT                                                                                             :              :            |
+        |           :   TEXT                                                                                             :              :            |
+        |           :   TEXT                                                                                             :              :            |
+        |           :   TEXT                                                                                             :              :            |
+        |           :   TEXT                                                                                             :              :            |
+      3 |_ _ _ _ _ _:_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ : _ _ _ _ _ _ _:_ _ _ _ _ _ |
+        |           :                                                                                                    :              :            |
+        |           :                                          [EXPAND / COLLAPSE]                                       :              :            |
+      4 |_ _ _ _ _ _:_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ : _ _ _ _ _ _ _:_ _ _ _ _ _ |
+        |           :                                                                                                    :              :            |
+        |           :                                          [Message Reactions]                                       :              :            |
+      5 |___________:____________________________________________________________________________________________________:______________:____________|
+        */
+        .messagebox .messagebox-content {
+            grid-template-rows: 25px repeat(3, auto);
+            padding-top: 2px;
+
+            .message_content {
+                padding: 0;
```

Comment:
I noticed this as a visual inconsistency between messages with and without senders: `.include-sender` shouldn’t change the _bottom_ padding of the content.

Reply:
Oh, yeah! Fixed to only modify `padding-top:0` for the `include-sender` case.